tests/dom-utils.test.js

```diff
@@ -1,129 +1,131 @@
-import { jest } from '@jest/globals';
-
+/* eslint accessor-pairs: ["error", {"setWithoutGet": false}] */
```

Comment:
I would prefer to maintain a `get` method rather to need to know what does this comment. It would be just:

```js
  get: jest.fn(() => $element.children),
```